### PR TITLE
Cultist

### PR DESCRIPTION
**Pull request by Danimal696**
Good for now, but ill release a revised version for weapon placement

### CODE REVIEW

File: CREDITS [hunk 1]
```diff
@@ -253,6 +254,7 @@ Claimed_*.mesh                         Skorpio                 CC-BY-SA 3.0, GPL
 Cre_vision_indicator.mesh              Andrew Buck             CC-BY-SA 3.0
 Crossbow.*                             Lamoot                  CC-BY 3.0               http://opengameart.org/content/low-poly-crossbow
 Crypt.mesh                             Danimal                 CC-BY-SA 3.0            http://forum.freegamedev.net/viewtopic.php?p=60771#p60771
+-> Cultist.mesh                        Danimal,Dread Knight,Katarzyna Zalecka                 CC-BY-SA 3.0                     http://opengameart.org/content/dark-priest
```
Comment (Bertram25) [inline CREDITS]: Why an arrow here?

Comment (Danimal696) [inline CREDITS]: copy paste error, sorry

**Comment by hwoarangmy** [comment]
Tested and it works well. Wanna change the arrow before merge ?

**Comment by hwoarangmy** [comment]
perfect

**Comment by akien-mga** [comment]
Did we make a creature config entry for it, or not yet? Should be done before 0.7.0 I guess.

**Pull request merged**